rasa/shared/utils/cli.py

```diff
@@ -5,7 +5,16 @@
 
 
 def print_color(*args: Any, color: Text):
-    print(rasa.shared.utils.io.wrap_with_color(*args, color=color))
+    output = rasa.shared.utils.io.wrap_with_color(*args, color=color)
+    try:
+        # colorama is used to fix a regression where colors can not be printed on
+        # windows. https://github.com/RasaHQ/rasa/issues/7053
+        from colorama import AnsiToWin32
```

Comment:
do we need this for other print commands such as `raise_warning` as well?

and do we need a `init(wrap=False)` call as described in their [documentation](https://pypi.org/project/colorama/) for `init(wrap=True)`?

Reply:
right good point, will add it to raise_warning as well. `init(...)` isn't needed, it doesn't do anything if wrap is set to `False` 😂

Reply:
I looked into the `raise_Warning` call and we can't add this there: colorama uses system calls to color text on a tty stream, which this isn't. 

I also don't think it is an issue there as warnings should handle the encoding issues - but we can still intervene if we see issues popping up on sentry

Reply:
sounds good. 

> init(...) isn't needed, it doesn't do anything if wrap is set to False 😂

I was wondering about that when reading their documentation 😂